### PR TITLE
[OAP-1631][Doc]Modify docs for OAP 0.9

### PR DESCRIPTION
**Pull request by HongW2019**

### CODE REVIEW

**Comment by zhixingheyi-tian** [comment]
@yao531441  Please have a review

File: docs/OAP-Installation-Guide.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# OAP Installation Guide
+This document introduces how to install OAP and its dependencies on your cluster nodes by **Conda** . Certain libraries need to be compiled and installed on your system using ***root*** account. 
+Follow steps below on ***every node*** of your cluster to set right environment for each machine.
+
+## Contents
+  - [Prerequisites](#prerequisites)
+  - [Compiling OAP](#compiling-oap)
```
Comment (yao531441) [inline docs/OAP-Installation-Guide.md:7]: ‘Compiling OAP’  how about install OAP

File: oap-cache/oap/docs/Developer-Guide.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCPACK_GENERATOR=rpm
 make package
 sudo rpm -i libvmemcache*.rpm
 ```
-##### build and install plasma
+#### build and install plasma
 To use optimized Plasma cache with OAP, you need following components:
```
Comment (yao531441) [inline oap-cache/oap/docs/Developer-Guide.md:76]: 'To use optimized Plasma cache with OAP, you need following components which will be installed by Conda': ?

File: docs/OAP-Installation-Guide.md (hunk 1 above)
Comment (HongW2019) [reply docs/OAP-Installation-Guide.md]: Modified.

File: oap-cache/oap/docs/Developer-Guide.md (hunk 2 above)
Comment (HongW2019) [reply oap-cache/oap/docs/Developer-Guide.md:76]: This is not applicable for this doc.

**Pull request merged**